lib/hukueki.py

```diff
@@ -0,0 +1,75 @@
+"""
+ねぇ、将来に何してるだろうね
+服役はしてないと良いね
+困らないでよ
+"""
+import discord
+
+
+HUKUEKI_MESSAGES = {
+"hukueki": """\
+```
+ねぇ、将来何してるだろうね
+{}はしてないと良いね
+困らないでよ
+```
+""",
+
+"lolicon": """\
+```
+だから僕は{}を辞めた - {} (Music Video)
+```
+"""
```

Comment:
```suggestion
"hukueki": "```\n" +
           "ねぇ、将来何してるだろうね\n" +
           "{}はしてないと良いね\n" +
           "困らないでよ\n" +
           "```",
"lolicon": "```だから僕は{}を辞めた - {} (Music Video)```"
```